pkg/database/authorized_app.go

```diff
@@ -77,7 +81,7 @@ func (db *Database) ListAuthorizedApps(includeDeleted bool) ([]*AuthorizedApp, e
 
 // CreateAuthorizedApp generates a new APIKey and assigns it to the specified
 // name.
-func (db *Database) CreateAuthorizedApp(name string, apiUserType APIUserType) (*AuthorizedApp, error) {
+func (db *Database) CreateAuthorizedApp(name string, apiUserType APIUserType, realm *Realm) (*AuthorizedApp, error) {
```

Comment:
I think it makes sense to standardize that realm is the first param when a realm is required.

Reply:
Also, why require a full Realm instead of just the ID?